src/main/java/org/helpberkeley/memberdata/Topic.java

```diff
@@ -24,9 +24,9 @@
 
 public class Topic {
     private final String name;
-    private final int id;
+    private final long id;
```

Comment:
This ID comes from Discourse and is described in their [API documentation](https://docs.discourse.org/) as an integer.  But in JSON, integers can be arbitrarily long.  So while it is unlikely that we will exceed 2^32 of anything on Discourse, it seemed better to use a long than an int for this.